drift_dev/lib/src/writer/writer.dart

```diff
@@ -198,7 +198,9 @@ abstract class _NodeOrWriter {
     final buffer = StringBuffer();
 
     for (final lexeme in code.elements) {
-      if (lexeme is DartTopLevelSymbol) {
+      if (lexeme is AnnotatedDartCode) {
```

Comment:
An `AnnotatedDartCode` shouldn't have another `AnnotatedDartCode` as an element, that's probably a bug in another part of the generator.

I think we should add an assertion to the constructor of `AnnotatedDartCode` checking that each element is either a `String` or a `DartTopLevelSymbol`. Then we can figure out where the wrong element comes from and fix that.

Reply:
That happens when the type converter is read: https://github.com/simolus3/drift/blob/develop/drift_dev/lib/src/writer/writer.dart#L115